android/src/main/java/com/courier/android/models/CourierPushNotificationEvent.kt

```diff
@@ -1,8 +1,6 @@
 package com.courier.android.models
 
-import com.google.firebase.messaging.RemoteMessage
-
 data class CourierPushNotificationEvent(
     val trackingEvent: CourierTrackingEvent,
-    val remoteMessage: RemoteMessage
+    val data: Map<String, String>
```

Comment:
I'd consider creating a class for the message data (ex. `CourierMessage` or something) that's more descriptive to pass around than a plain `Map` and let's us extend functionality down the line